src/System.IO.Pipelines/System/IO/Pipelines/WritableBuffer.cs

```diff
@@ -6,91 +6,17 @@
 
 namespace System.IO.Pipelines
```

Comment:
1. Change the name of this file.
2. Can we move this method to System.Buffers.Primitives?